### ISSUE TITLE
Search history

### ISSUE THREAD

**Issue by rgrinberg** [eugen0329/vim-esearch#196]
Is there to browse through the search history in this plugin? `C-p` and `C-n` do not work as one would expect.

**Comment by eugen0329** [comment]
You can use `<up>` and `<down>` to do so. `<c-p>` and `<c-n>` are mapped to use multipler patterns (e.g. `rg` with multiple `-e` options).
There's a pull request aimed to improve maintainability, so it'll be possible to remap them very soon.

Also I see that `<c-p>` and `<c-n>` is expected to be used more often than multi-pattern search, so you're welcome to suggest better defaults.

**Comment by eugen0329** [comment]
You can also remap `<c-j>` and `<c-k>` for a while
```vim
  cnoremap <c-j> <down>
  cnoremap <c-k> <up>
```

I even forgot that `<c-p>` is normally used to scroll through the history by default lol.

**Comment by rgrinberg** [comment]
Yes, I would recommend not to override default bindings. I would also love to have the `M-p` and `M-n` working as well. Which are supposed to search through the history backwards & forwards with the current input as the search key.
